int16 LED1
int16 LED2
int16 LED3

